src/Dev/Tasks/SecureAssetsMigrationHelper.php

```diff
@@ -138,14 +138,14 @@ public function setLogger(LoggerInterface $logger)
     public function htaccessMatch($content)
     {
         $regexes = $this->htaccessRegexes;
-        $lines = explode("\n", $content);
+        $lines = explode("\n", $content ?? '');
 
-        if (count($lines) != count($regexes)) {
+        if (count($lines ?? []) != count($regexes ?? [])) {
             return false;
         }
 
         foreach ($lines as $i => $line) {
-            if (!preg_match($regexes[$i], $line)) {
+            if (!preg_match($regexes[$i] ?? '', $line ?? '')) {
```

Comment:
```suggestion
            if (!preg_match($regexes[$i] ?? '//', $line ?? '')) {
```
`preg_match`, `preg_replace`, and probably other regex functions can't take an empty string for the regex argument. You end up getting
> [Warning]: preg_match(): Empty regular expression

Reply:
> [Warning]: preg_match(): Empty regular expression

IMO this is a better outcome than passing in a legit made up regex such as `//` which is almost a logic change.

We'd almost be better off not using the null collascing operator at all and which would risk null values here and potentially deprecation warnings.  I think I'd rather have a regular `Warning` over a `deprecation warning` though, as deprecated warnings in CI are very easier to ignore.

Reply:
Yup that's a fair point, I can get behind that logic.